src/chrome/content/rules/NASA.gov.xml

```diff
@@ -558,7 +558,6 @@
 	<target host="neba.nasa.gov" />
 	<target host="towerfts.nsbf.nasa.gov" />
 	<target host="*.towerfts.nsbf.nasa.gov" />
-	<target host="nasasearch.nasa.gov" />
```

Comment:
`dig @8.8.8.8 nasasearch.nasa.gov` works for me

Reply:
fixed. thanks!